src/ducks/account/AccountSwitch.styl

```diff
@@ -53,10 +53,10 @@
     position fixed
     top 78px
     left 0
-    bottom 10px
     width 440px
     overflow auto
     z-index: 60
+    max-height: calc(100vh - 78px)
```

Comment:
would be good to store the 78px in a variable for clarity

Reply:
I used [property lookup](http://stylus-lang.com/docs/variables.html#property-lookup), is it OK for you ?